packages/framework/esm-globals/src/types.ts

```diff
@@ -302,19 +302,15 @@ export type WorkspaceDefinition = {
    * The width "extra-wide" is for workspaces that contain their own sidebar.
    */
   width?: 'narrow' | 'wider' | 'extra-wide';
+  preferredWindowSize?: WorkspaceWindowState;
   /**
-   * Controls whether the workspace has its own sidebar. If true, the sidebar will be displayed when the workspace is open.
-   */
-  hasOwnSidebar?: boolean;
-  /**
-   * Sidebars have icons that representing workspaces. The sidebar family is the name of the
-   * sidebar that should contain the icon for this workspace. This is generally only needed if
-   * `hasOwnSidebar` is true, in which case this is the name of that sidebar. If multiple
-   * workspaces have `hasOwnSidebar` set to true and the same family name, then the sidebar
-   * within the workspace area will have icons for each of those workspaces.
+   * Workspaces can open either individually or in a group of workspaces. The workspace groups
+   * will define the groups in which a workspace can be opened.
+   *
+   * In case the currently opened workspace is not present in the groups of a workspace,
+   * the current group will close and then the workspace will be launched.
```

Comment:
I feel like the description here is unclear. Maybe:

```suggestion
   * Workspaces can open either independently or as part of a "workspace group". A
   * "workspace group" groups related workspaces together, so that only one is visible
   * at a time. For example, <USE CASE DESCRIPTION HERE>.
   *
   * In case the currently opened workspace is not present in the groups of a workspace,
   * the current group will close and then the workspace will be launched.
```